src/common/datetime.cljs

```diff
@@ -0,0 +1,258 @@
+(ns common.datetime
```

Comment:
imo we should some for of "common" folder at the root for utilities like date time and i18n etc.

Reply:
please ignore 😓  🙃